cfpb/consumerfinance.gov
# TCCP: Renaming the app and filter logic updates

This PR renames the prototype TCCP app from "Find cards for your situation" to "Explore cards for your situation". It therefore moves the URL to http://localhost:8000/consumer-tools/credit-cards/explore-cards/.

This PR also iterates on the TCCP filtering logic:

- We no longer filter on whether introductory APR is offered
- We no longer filter on secured cards
- Ordering by purchase APR now uses a fallback of product name to break ties
- Ordering by transfer APR now uses the coalesce of the tier-specific transfer APR OR the transfer APR min; similarly, when displaying transfer APR on the results view we now show a range if that's what we have
- Our situation logic has been changed in accordance with the above

## How to test this PR

To test, run a local server and visit http://localhost:8000/consumer-tools/credit-cards/explore-cards/.

Or, visit internal testing DEV4 server to see this PR deployed there.

## Checklist
